protos/mocap/mocap.proto

```diff
@@ -51,7 +52,75 @@ message AttPosMocap {
+      MAV_FRAME_GLOBAL_RELATIVE_ALT = 3; // Global (WGS84) coordinate frame + altitude relative to the home position.
+      MAV_FRAME_LOCAL_ENU = 4; // Local coordinate frame, Z-up (x: east, y: north, z: up).
+      MAV_FRAME_GLOBAL_INT = 5; // Global (WGS84) coordinate frame (scaled) + MSL altitude.
+      MAV_FRAME_GLOBAL_RELATIVE_ALT_INT = 6; // Global (WGS84) coordinate frame (scaled) + altitude relative to the home position.
+      MAV_FRAME_LOCAL_OFFSET_NED = 7; // Offset to the current local frame. Anything expressed in this frame should be added to the current local frame position.
+      MAV_FRAME_BODY_NED = 8; // Setpoint in body NED frame. WARNING: DEPRECATED: Replaced by MAV_FRAME_BODY_FRD (2019-08).
+      MAV_FRAME_BODY_OFFSET_NED = 9; // Offset in body NED frame. WARNING: DEPRECATED: Replaced by MAV_FRAME_BODY_FRD (2019-08).
+      MAV_FRAME_GLOBAL_TERRAIN_ALT = 10; // Global (WGS84) coordinate frame with AGL altitude (at the waypoint coordinate).
+      MAV_FRAME_GLOBAL_TERRAIN_ALT_INT = 11; // Global (WGS84) coordinate frame (scaled) with AGL altitude (at the waypoint coordinate).
+      MAV_FRAME_BODY_FRD = 12; // Body fixed frame of reference, Z-down (x: forward, y: right, z: down).
+      MAV_FRAME_BODY_FLU = 13; // Body fixed frame of reference, Z-up (x: forward, y: left, z: up).
+      MAV_FRAME_MOCAP_NED = 14; // Odometry local coordinate frame of data given by a motion capture system, Z-down (x: north, y: east, z: down). WARNING: DEPRECATED: Replaced by MAV_FRAME_LOCAL_FRD (2019-08).
+      MAV_FRAME_MOCAP_ENU = 15; // Odometry local coordinate frame of data given by a motion capture system, Z-up (x: east, y: north, z: up). WARNING: DEPRECATED: Replaced by MAV_FRAME_LOCAL_FLU (2019-08).
+      MAV_FRAME_VISION_NED = 16; // Odometry local coordinate frame of data given by a vision estimation system, Z-down (x: north, y: east, z: down). WARNING: DEPRECATED: Replaced by MAV_FRAME_LOCAL_FRD (2019-08).
+      MAV_FRAME_VISION_ENU = 17; // Odometry local coordinate frame of data given by a vision estimation system, Z-up (x: east, y: north, z: up). WARNING: DEPRECATED: Replaced by MAV_FRAME_LOCAL_FLU (2019-08).
+      MAV_FRAME_ESTIM_NED = 18; // Odometry local coordinate frame of data given by an estimator running onboard the vehicle, Z-down (x: north, y: east, z: down). WARNING: DEPRECATED: Replaced by MAV_FRAME_LOCAL_FRD (2019-08).
+      MAV_FRAME_ESTIM_ENU = 19; // Odometry local coordinate frame of data given by an estimator running onboard the vehicle, Z-up (x: east, y: noth, z: up). WARNING: DEPRECATED: Replaced by MAV_FRAME_LOCAL_FLU (2019-08).
+      MAV_FRAME_LOCAL_FRD = 20; // Forward, Right, Down coordinate frame. This is a local frame with Z-down and arbitrary F/R alignment (i.e. not aligned with NED/earth frame).
+      MAV_FRAME_LOCAL_FLU = 21; // Forward, Left, Up coordinate frame. This is a local frame with Z-up and arbitrary F/L alignment (i.e. not aligned with ENU/earth frame).
```

Comment:
Can we try not to expose all the MAVLink baggage (aka all the unused frames) and only expose a supported and very clear subset?

Reply:
Hm.
This is good idea. But I'm afraid that some systems or users can use or maintain old revisions of PX4. 
Is it not critical?

Reply:
We only need to support the current release and master usually. For the future we might want to be somewhat backwards compatible but for now I would not worry about that, especially since it's a new feature in MAVSDK anyway.

Reply:
Done: https://github.com/mavlink/MAVSDK-Proto/pull/105/commits/624e16b4e2213a3fd244c350236c20ed8dc45e94